datumaro/plugins/yolo_format/extractor.py

```diff
@@ -572,6 +611,13 @@ def make_children_names(skeleton_label):
 
         return categories
 
+    def _map_label_id(self, ann_label_id):
+        skeleton_id = super()._map_label_id(ann_label_id)
+        label_id = self._skeleton_id_to_label_id.get(skeleton_id, -1)
+        if self._categories[AnnotationType.label][label_id].parent != "":
+            raise InvalidAnnotationError("WTF")
```

Comment:
Please make the message more descriptive.

Reply:
Removed it. This error is more of a debug artefact, I think this situation can not happen